src/Query/QueryResult.php

```diff
@@ -485,13 +485,35 @@ public function toArray() {
 			'meta'=> [
 				'hash'   => md5( json_encode( $serializeArray ) ),
 				'count'  => $this->getCount(),
+				'total'  => $this->getTotalCount(),
 				'offset' => $this->mQuery->getOffset(),
 				'source' => $this->mQuery->getQuerySource(),
 				'time'   => number_format( ( microtime( true ) - $time ), 6, '.', '' )
 				]
 			]
 		);
 	}
+	
+	/**
+	 * Returns the total number of query results regardless of maximum to retrieval
+	 * 
+	 * @since ...
+	 *
+	 * @return integer|null
+	 */
+	public function getTotalCount() {
+		$query = $this->mQuery;
+		$query->querymode = \SMWQuery::MODE_COUNT; //change query format to count
```

Comment:
This side-effect is not undone, so calls to other methods might no longer be correct once you call this new method